package.json

```diff
@@ -754,5 +760,23 @@
       "@metamask/solana-wallet-snap>@solana/web3.js>bigint-buffer": false
     }
   },
-  "packageManager": "yarn@4.5.1"
+  "packageManager": "yarn@4.5.1",
+  "foundryup": {
+    "binaries": [
+      "anvil"
+    ],
+    "checksums": {
+      "algorithm": "sha256",
+      "binaries": {
+        "anvil": {
+          "darwin-amd64": "6ad1c31bb8bc8c78551792ad538028385eb5a44d105f9af40ddeb1b75c1327b1",
+          "darwin-arm64": "cbb36860b2e08a7d5a35bec430904a1dc0319602ac0923017b81d9a8ffdcc5b0",
+          "linux-amd64": "56054e7d59547beb6adacbb9060a58595a8c5cca185f946317334ece509e0433",
+          "linux-arm64": "16494ae631d8ef73944abaaaa756661c1d43bbfc7642e7fff0a0d807d6d143c4",
+          "win32-amd64": "2034bdf35023e59c5dc06d6717a8eb2cede1204dbc60fd2507564422f03dceb3"
+        }
+      }
+    },
+    "version": "nightly-31dd1f77fd9156d09836486d97963cec7f555343"
```

Comment:
foundry doesn't publish versions, just tags like this. it's weird, but it's all we got.